src/import/importd.c

```diff
@@ -111,6 +111,8 @@ struct Manager {
 
         bool use_btrfs_subvol;
         bool use_btrfs_quota;
+
+        RuntimeScope runtime_scope; /* for now: always RUNTIME_SCOPE_SYSTEM */
```

Comment:
Maybe better to set `RUNTIME_SCOPE_SYSTEM` explicitly in `manager_new()`, like others do.